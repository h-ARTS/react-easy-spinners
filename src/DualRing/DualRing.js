import React from 'react'
import styles from './styles.css'

const DualRing = (props) => (
  <>
    <style>${styles}</style>
    <div className="lds-dual-ring"></div>
  </>
)

export default DualRing
