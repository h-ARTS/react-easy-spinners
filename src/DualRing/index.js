export { default as DualRing } from './DualRing'
