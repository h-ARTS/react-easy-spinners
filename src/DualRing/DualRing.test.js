import React from 'react'
import renderer from 'react-test-renderer'
import DualRing from './DualRing'

describe('DualRing component', () => {
  test('should render correctly', () => {
    const tree = renderer.create(<DualRing />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
