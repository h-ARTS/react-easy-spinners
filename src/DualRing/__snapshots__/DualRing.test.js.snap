// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DualRing component should render correctly 1`] = `
<div
  className="lds-dual-ring"
/>
`;
